wingtips-core/src/test/java/com/nike/wingtips/http/HttpRequestTracingUtilsTest.java

```diff
@@ -260,6 +260,51 @@ public void fromRequestWithHeaders_sets_sampleable_to_attribute_value_if_sampled
         assertThat(newSpan.isSampleable()).isFalse();
     }
 
+    @DataProvider(value = {
```

Comment:
did you want to add a special case test that wingtips coerces lack of sampled flag as sampled=true? This test would change if/when there's a trace id sampler impl here.